specification/vmware/Microsoft.AVS/routes.tsp

```diff
@@ -0,0 +1,1362 @@
+  @get
+  get is Azure.Core.Foundations.Operation<
+    {
+      ...SubscriptionIdParameter;
+      ...ResourceGroupParameter;
+      ...PrivateCloudNameParameter;
+      ...ClusterNameParameter;
+      ...VirtualMachineIdParameter;
+    },
+    VirtualMachine
+  >;
+
+  @doc("Enable or disable DRS-driven VM movement restriction")
+  @route("/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AVS/privateClouds/{privateCloudName}/clusters/{clusterName}/virtualMachines/{virtualMachineId}/restrictMovement")
+  @post
+  restrictMovement is Azure.Core.Foundations.Operation<
+    {
+      ...SubscriptionIdParameter;
+      ...ResourceGroupParameter;
+      ...PrivateCloudNameParameter;
+      ...ClusterNameParameter;
+      ...VirtualMachineIdParameter;
+
+      @doc("Whether VM DRS-driven movement is restricted (Enabled) or not (Disabled)")
+      @body
+      restrictMovement: VirtualMachineRestrictMovement;
+    },
+    void
+  >;
+}
+
+interface PlacementPolicies {
+  @doc("List placement policies in a private cloud cluster")
+  @route("/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AVS/privateClouds/{privateCloudName}/clusters/{clusterName}/placementPolicies")
+  @get
+  list is Azure.Core.Foundations.Operation<
+    {
+      ...SubscriptionIdParameter;
+      ...ResourceGroupParameter;
+      ...PrivateCloudNameParameter;
+      ...ClusterNameParameter;
+    },
+    PlacementPoliciesList
+  >;
+
+  @doc("Get a placement policy by name in a private cloud cluster")
+  @route("/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AVS/privateClouds/{privateCloudName}/clusters/{clusterName}/placementPolicies/{placementPolicyName}")
+  @get
+  get is Azure.Core.Foundations.Operation<
+    {
+      ...SubscriptionIdParameter;
+      ...ResourceGroupParameter;
+      ...PrivateCloudNameParameter;
+      ...ClusterNameParameter;
+      ...PlacementPolicyNameParameter;
+    },
+    PlacementPolicy
+  >;
+
+  @doc("Create or update a placement policy in a private cloud cluster")
+  @route("/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AVS/privateClouds/{privateCloudName}/clusters/{clusterName}/placementPolicies/{placementPolicyName}")
+  @put
+  createOrUpdate is Azure.Core.Foundations.Operation<
+    {
+      ...SubscriptionIdParameter;
+      ...ResourceGroupParameter;
+      ...PrivateCloudNameParameter;
+      ...ClusterNameParameter;
+      ...PlacementPolicyNameParameter;
+
+      @doc("A placement policy in the private cloud cluster")
+      @body
+      placementPolicy: PlacementPolicy;
+    },
+    PlacementPolicy
+  >;
+
+  @doc("Update a placement policy in a private cloud cluster")
+  @route("/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AVS/privateClouds/{privateCloudName}/clusters/{clusterName}/placementPolicies/{placementPolicyName}")
```

Comment:
In geenral, you should never have `@route` in an ARM typespec.  If your resources are modeled as parent/children, you should be able to use the standard operation templates